### PR TITLE
EKS docs page updated with content

### PR DESCRIPTION
**Pull request by senpx**

### CODE REVIEW

File: _includes/k8s-spec-generate.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ To generate the spec file, head on to the below URLs for the PX release you wish
 
 Alternately, you can use curl to generate the spec as described in [Generating Portworx Kubernetes spec using curl](/scheduler/kubernetes/px-k8s-spec-curl.html).
 
+Note: For the EKS clusters, select the EKS checkbox option when we create the Portworx spec.
```
Comment (harsh-px) [inline _includes/k8s-spec-generate.md]: change "we" to "you".

File: cloud/aws/eks.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+![k8s porx Logo](/images/k8s-porx.png){:height="188px" width="188px"}
+
 * TOC
 {:toc}
 
-## Coming soon
+
+## Create a EKS cluster
+
+Creating EKS clusters can be found [here](https://aws.amazon.com/eks/).
```
Comment (harsh-px) [inline cloud/aws/eks.md]: Incomplete sentence. Should be "Instructions on creating EKS ..."

File: cloud/aws/eks.md [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+
+The steps below will help you enable dynamic provisioning of Portworx volumes in your Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster.
```
Comment (harsh-px) [inline cloud/aws/eks.md]: Don't repeat Amazon. So just have EKS in the brackets.

Comment (senpx) [reply cloud/aws/eks.md]: Updating it but that's how Amazon itself posting in their site. https://aws.amazon.com/eks/

Comment (harsh-px) [reply cloud/aws/eks.md]: Sorry, in that case, let's maintain their branding. You can keep what you had before.

**Comment by harsh-px** [comment]
Also fix the CI build.

**Pull request merged**